Tools/src/coverage.sh

```diff
@@ -30,4 +30,4 @@ done
 go tool cover -html="$profile" -o ./bin/coverage.html
 go tool cover -func="$profile"
 
-rm -rf "$work_dir"
\ No newline at end of file
+rm -rf "$work_dir"
```

Comment:
It's the addition of a new line at the end of the file. i can get rid of it if you'd like.